app/src/main/java/com/iti/android/team1/ebuy/ui/register_screen/view/RegisterScreen.kt

```diff
@@ -102,6 +102,8 @@ class RegisterScreen : Fragment() {
             binding.edtFirstName.text.toString().trim(),
             binding.edtLastName.text.toString().trim(),
             binding.edtPassword.text.toString().trim())
+        "noobmaster@lod.com"
+        "ASDASsdD##$123"
```

Comment:
what's this?

Reply:
l left them to make it easy in you'd we sign in once more,

Reply:
oh forgot to delete them I thought that u mean the XML text

Reply:
No, remove it from the code!

Reply:
I just said that "oh forgot to delete them I thought that u mean the XML text"